test/functional/ui/searchhl_spec.lua

```diff
@@ -25,20 +25,6 @@ describe('search highlighting', function()
     screen:set_default_attr_ignore( {{bold=true, foreground=hl_colors.NonText}} )
   end)
 
-  it('is off by default', function()
-    insert("some text\nmore text")
-    feed("gg/text<cr>")
-    screen:expect([[
-      some ^text                               |
-      more text                               |
-      ~                                       |
-      ~                                       |
-      ~                                       |
-      ~                                       |
-      /text                                   |
-    ]])
-  end)
```

Comment:
I made something slightly different, I turned this spec into a test for `:set nohlsearch`.